Please fix 

1s
Run markdownlint-cli2
markdownlint-cli2 v0.23.1 (markdownlint v0.41.1)
Finding: **/*.md **/*.markdown !target/**
Linting: 108 files
Summary: 1 issue in 1 file
docs/archive/pr-summaries/pr-summary-357.md:47 error MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]
Error: Process completed with exit code 1.

--- a/docs/archive/pr-summaries/pr-summary-357.md
+++ b/docs/archive/pr-summaries/pr-summary-357.md
@@ -44,7 +44,7 @@ flowchart LR
 Backend/test-only change — no web interface to screenshot. The GPU tests run on
 this machine (Metal adapter present, not skipped):
 
-```
+```text
 running 3 tests
 test reused_bind_group_preserves_cpu_parity ... ok
 test same_size_chunks_are_deterministic_and_reuse_bind_groups ... ok

